mantidproject/mantid
# Reduce pylint warnings in reflectometer files

**To Test**:
- Check that pylint warnings for Reflectometer files has been reduced from 707 on pylint build
- Simple code review
- [x] refl_gui.py (previously 56 violations)
- [x] base_ref_reduction.py (previously 77 violations)
- [x] wks_ultility.py (previously 73 violations)
- [x] refm_reduction.py (previously 57 violations)

Fixes #14702
